src/logic/basic.lean

```diff
@@ -376,6 +378,9 @@ protected lemma eq.congr {x₁ x₂ y₁ y₂ : α} (h₁ : x₁ = y₁) (h₂ :
   (x₁ = x₂) ↔ (y₁ = y₂) :=
 by { subst h₁, subst h₂ }
 
+lemma eq.congr_left {x y z : α} (h : x = y) : x = z ↔ y = z := by rw [h]
```

Comment:
How do you use this lemma?

Reply:
`rw` is not always useful, because the proof `h` might be multiple lines long.
`congr` just fails.
I didn't try `congr' 2`, which does work and does the same thing. 

Still I think it's useful to have a lot of basic lemmas like this. Not in the least because it is sometimes easier to apply a lemma than to see which tactic applies (especially in a case where the most obvious tactic, `congr`, fails).